### ISSUE TITLE
Running icommands with new user throws errors but still works

### ISSUE THREAD

**Issue by K41eb** [irods/irods#5447]
## Bug Report

Low severity?

### iRODS Version, OS and Version

iRODS 4.2.8
CentOS 7

### What did you try to do?

I created a brand new "bobby" user to test federation. Upon running `iinit` and any subsequent `icommand` the command always prints:
```
 ERROR: resolve_hostname_from_hosts_config :: caught exception:
<no exception message / code / trace logs>
<expected output>
```
### Expected behavior

There should be no error message, or if indeed there is an error, a more detailed message.

### Observed behavior (including steps to reproduce, if applicable)

1. Fresh install
2. `iadmin mkuser bobby rodsuser`
3. `iadmin moduser bobby password bobby`
4. Switch to user bobby
5. `iinit`, `ils`, `iput`, `iget`, `ils /REMOTE_ZONE/home/rods/public` ... any `icommand` really.
 
` ERROR: resolve_hostname_from_hosts_config :: caught exception:` is systematically printed before the command's output, even when there's none.

**Comment by trel** [comment]
@korydraughn can you reproduce this?

**Comment by korydraughn** [comment]
I have not been able to reproduce this. The commands are working as expected. I tried this using docker-compose with ubuntu 16 containers.

@K41eb Can you provide any other information about your environment? Are you using a hosts_config.json file? Do the icommands produce that message when use in the local zone?

**Comment by trel** [comment]
This issue was resolved in 4.2.9 by the following PR: #5403

Marking as resolved and closing
